OverPassLayer.js

```diff
@@ -229,6 +244,11 @@ L.OverPassLayer = L.FeatureGroup.extend({
           var queryWithMapCoordinates = this.options.query.replace(/(BBOX)/g, bbox.toOverpassBBoxString());
           var url =  this.options.endpoint + "interpreter?data=[out:json];" + queryWithMapCoordinates;
 
+          if (beforeRequest) {
```

Comment:
Yes. The thing is that I want to call the callback just once. I mean, the current BBOX is splitted in smaller ones and I want to call the `beforeCallback` just once, so I have to check if I already called it.